site/topics/logic/truth-tables.rst

```diff
@@ -2,14 +2,91 @@
 Truth Tables
 ************
 
+* Much of the following will likely be a review of already well understood concepts
+* This content is covered for completeness, but will be kept at a relatively high level
+
 
 
 Boolean Logic
 =============
 
+* Boolean logic is a form of algebra that operates on boolean values that can take on only two states
+* These states are typically called *True* and *False*
```

Comment:
I think of them as _true_ and _false_, and believe python to be very much in the wrong about the capitalization of their names.